manim/mobject/mobject.py

```diff
@@ -2287,6 +2287,28 @@ def construct(self):
             sm1.interpolate_color(sm1, sm2, 1)
         return self
 
+    def match_points(self, mobject: "Mobject", copy_submobjects: bool = True):
+        """Edit points, positions, and submobjects to be identical
+        to another :class:`~.Mobject`, while keeping the style unchanged.
+
+        Examples
+        --------
+        .. manim:: BecomeScene
```

Comment:
Ah, I think I found the issue. This should probably be `MatchPointsScene`, not `BecomeScene`
```suggestion
        .. manim:: MatchPointsScene
```

Reply:
Wow, can't believe I didn't spot that. Thanks!